otcextensions/tests/functional/sdk/dms/v1/test_instance.py

```diff
@@ -90,7 +89,9 @@ def tearDown(self):
         except openstack.exceptions.SDKException as e:
             _logger.warning('Got exception during clearing resources %s'
                             % e.message)
-        self._deinitialize_network()
+        finally:
```

Comment:
same as above, however if we need finally for _deinitialize - no problem

Reply:
In this case "finally" used mostly for _deinitialize, I think we should leave it as is.